fmralign/tests/test_pairwise_alignment.py

```diff
@@ -34,6 +43,26 @@ def test_pairwise_identity():
         assert_algo_transform_almost_exactly(
             algo, img1, img1, mask=mask_img)
 
+    # test intersection of clustering and mask
+    data_mask = copy.deepcopy(mask_img.get_data())
+    data_mask[0] = 0
+    # create ground truth
+    clustering_mask = new_img_like(mask_img, data_mask)
+    data_clust = copy.deepcopy(data_mask)
+    data_clust[1] = 3
+    # create 2-parcels clustering, smaller than background
+    clustering = new_img_like(mask_img, data_clust)
+
+    # clustering is smaller than mask
```

Comment:
Same question as above -- should we also have a test case where the parcellation is larger than the mask ?

Reply:
Actually its already done. Every clustering has at least 0s outside the mask which are removed by masking. But I'm always down for a new test so I will check if this would add something